src/hci/gnomex/controller/GNomExCommand.java

```diff
@@ -48,6 +55,26 @@
   public GNomExCommand() {
   }
 
+  public abstract Command executeCommand() throws Exception;
+
+  protected abstract Logger getLogger();
```

Comment:
+1  I like this template pattern.  Force all the concrete classes to implement getLogger.  Nice!